src/workspace-data/ImportStatus.ts

```diff
@@ -76,13 +93,20 @@ function ImportStatusItem({ job: { targetWorkspace, jobId }, onDone }) {
         ]),
       });
 
-    const errorNotify = () => notify('error', 'Error importing data.', { message });
+    const errorNotify = () => notify('error', 'Error importing data.', { notificationMessage });
+
+    // import service statuses: Pending, Translating, ReadyForUpsert, Upserting, Done, Error
```

Comment:
yeah the consensus seems to be that I should do that

Reply:
not sure if you are still working on changing this part or landed on a different path but I was sort of picturing this as a check if workspace is azure or GCP and then checking for status values and providing appropriate notify separately for each

Reply:
I'm not thrilled with the current implementation but I'm also not thrilled with checking the workspace either - everything here is the same except for the specific words so that extra check doesn't seem to add much